### PR TITLE
Experimental percentile

### LINKED ISSUE #10740: percentile

### PR DESCRIPTION
**Pull request by pradyyadav**
close #10740

### CODE REVIEW

**Comment by pradyyadav** [comment]
Hi @Fayad-Alman, I am not sure how to handle this issue.

image

**Comment by valeriocost** [comment]
Hello @pradyyadav 😄 I see there are a lot of comments/debug prints, could you clean the committed code? Thanks

**Comment by pradyyadav** [comment]
Hi @valeriocost , I would surely remove unnecessary code. However I am facing issues with the tests in local on jax and torch, saying that the output doesn't match, the problem is with the shape of output not matching and for few cases, the values aren't the same.

**Comment by valeriocost** [comment]
I'll try to run it locally tomorrow👍🏼

**Comment by valeriocost** [comment]
did you mean numpy and torch? The tests fail for me in those backends

**Comment by pradyyadav** [comment]
It is never fixed, I mostly get failed tests in jax and torch.

File: ivy/functional/backends/torch/experimental/statistical.py [hunk 1]
```diff
@@ -133,3 +133,79 @@ def bincount(
...
+    a: torch.Tensor,
+    q: Union[Sequence[float], float],
+    /,
+    *,
+    axis: Optional[Union[Sequence[int], int]] = None,
+    keepdims: bool = False,
+    interpolation: str = "linear",
+    out: Optional[torch.Tensor] = None,
+) -> torch.Tensor:
+    # if isinstance(axis, list) or isinstance(axis, tuple):
```
Comment (Daniel4078) [inline ivy/functional/backends/torch/experimental/statistical.py:152]: why are these comments here?

**Comment by valeriocost** [comment]
@pradyyadav Sorry for the late reply 😅 running the test locally does not show them as failed, what was the problem? Also, can you fix the formatting i.e. the comments as @Daniel4078 mentioned? Thanks 😄

**Comment by pradyyadav** [comment]
Hi @valeriocost , actually what happens is you may get passed cases for a few times, it is like pytest results keep changing all the time. I can give much detail on weekend

**Comment by valeriocost** [comment]
sure, no pressure 😄

**Comment by ivy-seed** [comment]
This PR has been labelled as stale because it has been inactive for more than 7 days. If you would like to continue working on this PR, then please add another comment or this PR will be closed in 7 days.

**Comment by ivy-seed** [comment]
This PR has been closed because it has been marked as stale for more than 7 days with no activity.

**Pull request closed without merging**